index.html

```diff
@@ -40,13 +43,13 @@ <h4 style="margin-top:-50px;">[auto saves every 3 seconds]</h4>
             }
             
             function wordCount() {
-                var count = document.querySelector("#note").innerHTML.trim().replace(/  +/g, ' ').split(' ').length;
-                if (document.querySelector("#note").innerHTML.length == 0) count = 0;
+                var count = noteHTMLSelector.innerHTML.trim().replace(/  +/g, ' ').split(' ').length;
+                if (noteHTMLSelector.innerHTML.length == 0) count = 0;
                 document.querySelector("#wordNum").innerHTML = "Word Count: " + count; 
             }
 
             function writeFile() {
-                fs.writeFile('/note', document.querySelector('#note').innerHTML, function (err) {
+                fs.writeFile('/note', noteHTMLSelector, function (err) {
```

Comment:
Shouldn't this be noteHTMLSeletor.innerHTML to reflect the content rather than the selector itself

```js
fs.writeFile('/note', noteHTMLSelector.innerHTML, function (err) {
```